packages/amplify-graphql-function-transformer/src/graphql-function-transformer.ts

```diff
@@ -195,3 +255,256 @@ const lambdaArnKey = (name: string, region?: string, accountId?: string): string
   // eslint-disable-next-line no-template-curly-in-string
   return `arn:aws:lambda:${region ? region : '${AWS::Region}'}:${accountId ? accountId : '${AWS::AccountId}'}:function:${name}`;
 };
+
+// #region Validation
+
+/**
+ * Validates that a `@function` directive is used in a supported way.
+ * @param config the {@link FunctionDirectiveConfiguration} to validate.
+ * @param definition the {@link FieldDefinitionNode} on which the `@function` directive is used.
+ * @param ctx the {@link TransformerContextProvider}
+ */
+const validate = (config: FunctionDirectiveConfiguration, definition: FieldDefinitionNode, ctx: TransformerContextProvider): void => {
+  if (config.invocationType === 'Event') {
+    // For any occurance  of 'invocationType: Event' validate:
```

Comment:
nit (future PR is OK)
```suggestion
    // For any occurence  of 'invocationType: Event' validate:
```

Reply:
Done. Addressed in https://github.com/aws-amplify/amplify-category-api/pull/2868